enable: true
elements:
  - id: rocks
    type: ScienceElementType.DISPLAY
    display_name: Display
    type_data:
      topic: "master_com/master_to_ros/x85"
      parser: send_weight
    buttons:
      - id: rocks_tare
        name: Tare
        message: MasterMessage.GS_TO_ESP
        data: [ 0, 0 ]
        color: yellow-foreground
      - id: rocks_request
        name: Request
        message: MasterMessage.GS_TO_ESP
        data: [ 0, 1 ]
        color: blue-foreground
  - id: container
    type: ScienceElementType.CONTAINER
    display_name: Container
    type_data:
      color: red-foreground
      open_message: MasterMessage.GS_TO_ESP
      open_data: [ 0 ]
      close_message: MasterMessage.GS_TO_ESP
      close_data: [ 1 ]
  - id: player
    type: ScienceElementType.PLAYER
    display_name: Player
    type_data:
      color: yellow-foreground
      play_message: MasterMessage.GS_TO_ESP
      play_data: [ 0, 0 ]
      pause_message: MasterMessage.GS_TO_ESP
      pause_data: [ 0, 1 ]
      stop_message: MasterMessage.GS_TO_ESP
      stop_data: [ 1, 1 ]
  - id: status
    type: ScienceElementType.STATUS
    display_name: Status
    type_data:
      topic: "master_com/master_to_ros/x85"
      parser: send_status
